controller/src/beerocks/master/db/db.cpp

```diff
@@ -19,6 +20,66 @@ using namespace beerocks_message;
 using namespace son;
 using namespace net;
 
+static const std::string timestamp_str            = "timestamp";
+static const std::string timelife_delay_str       = "timelife";
+static const std::string initial_radio_enable_str = "initial_radio_enable";
+static const std::string initial_radio_str        = "initial_radio";
+static const std::string selected_band_enable_str = "selected_band_enable";
+static const std::string selected_bands_str       = "selected_bands";
+
+// static
+std::string db::type_to_string(beerocks::eType type)
+{
+    if (type == beerocks::eType::TYPE_GW) {
+        return std::string("gateway");
```

Comment:
Why not use
```
return "gateway";
```
and not explicitly create `std::string` object?

Reply:
replaced with switch-case that return `"*"` instead of `std::string("*")`